examples/doc/SkeletalFinSet.g

```diff
@@ -10,7 +10,7 @@ Display( m );
 IsWellDefined( m );
 #! true
 String( m );
-#! "FinSet( SkeletalFinSets, 7 )"
+#! "ObjectConstructor( SkeletalFinSets, 7 )"
```

Comment:
I think we should keep the old names because those can be found in the documentation. More generally, we should discuss how the interface for the different models of (skeletal) finite sets should look like and how we document it. For example, I think we should get rid of `FinSetNC` in favor of `ObjectConstructor` for non-skeletal finite sets, but we currently do not have a standard way for documenting `ObjectConstructor` I think.

Reply:
Done. I had to touch other places in the code and another example.